### PR TITLE
feat: Bumped integral_isolates package for flame_isolate

### PR DESCRIPTION
**Pull request by lohnn**
# Description

Updated package dependencies for flame_isolate.
Added support for TailoredIsolate and computeStream (called isolateComputeStream in flame_isolate).

Updated the example package with new sprites created by yours truly, and updated the license for the package, now that there is no assets with copy-left CC license remaining.

## Checklist

- [x] I have followed the [Contributor Guide] when preparing my PR.
- [x] I have updated/added tests for ALL new/updated/fixed functionality.
- [x] I have updated/added relevant documentation in `docs` and added dartdoc comments with `///`.
- [x] I have updated/added relevant examples in `examples` or `docs`.

## Breaking Change?

- [ ] Yes, this PR is a breaking change.
- [x] No, this PR is not a breaking change.

### Migration instructions

Computation function is renamed from `isolate` to `isolateCompute`.
There are future breaking changes that are, as of now, mitigated with a `@Deprecated` annotation with information regarding what function to use instead.

### CODE REVIEW

File: packages/flame_isolate/lib/flame_isolate.dart [hunk 1]
```diff
@@ -6,8 +6,9 @@ export 'package:integral_isolates/integral_isolates.dart';
 /// Mixin on [Component] that holds an instance of a long running isolate using
 /// the library integral_isolates.
 ///
-/// Using isolate is done by just running [isolate] function the same way you
-/// would run Flutter's compute function.
+/// Using isolate is done by just running [isolateCompute] function the same way
```
Comment (spydon) [inline packages/flame_isolate/lib/flame_isolate.dart:9]:
```suggestion
/// Using the isolate is done by just running [isolateCompute] function the same way
```

File: packages/flame_isolate/lib/flame_isolate.dart [hunk 2]
```diff
@@ -50,12 +51,41 @@ mixin FlameIsolate on Component implements IsolateGetter {
     _isolate = null;
   }
 
-  @override
+  /// Deprecated in favor of [isolateCompute].
+  @Deprecated('Use isolateCompute instead.')
```
Comment (spydon) [inline packages/flame_isolate/lib/flame_isolate.dart:55]: Write which version it will be removed in, v0.6.0 I guess?

**Review by spydon — APPROVED** [review]
Lgtm! Should be a `feat` since it exposes new methods.

File: packages/flame_isolate/lib/flame_isolate.dart (hunk 2 above)
Comment (lohnn) [reply packages/flame_isolate/lib/flame_isolate.dart:55]: Well, it is already removed in the underlying package, so this is a soft deprecation that can be removed in the flame_isolate package whenever.

Hence a bit up to you and the best practices of Flame :)

Comment (spydon) [reply packages/flame_isolate/lib/flame_isolate.dart:55]: v0.7.0 then, since this one will be 0.6.0 now due to the `feat` :)

**Review by spydon — APPROVED** [review]

**Pull request merged**